endToEndTests/setup.go

```diff
@@ -152,6 +162,11 @@ func getS3File(region, bucket, filename string, decrypt bool) (io.ReadCloser, er
 			return nil, err
 		}
 
+		log.Debug("reading file with psk",
+			log.Data{"vault_path": cfg.VaultPath,
+				"filename": filename,
+				"psk": psk})
```

Comment:
Could you change this to be: 


``` go
"psk": hex.DecodeString(psk)
```

 just to make it more readable?

Reply:
Done